operator/v1/types_network.go

```diff
@@ -480,6 +486,14 @@ type HybridOverlayConfig struct {
 type IPsecConfig struct {
 }
 
+type IPsecExternal struct {
+	// State controls the node's external (aka NS) ipsec service state.
+	// +kubebuilder:validation:Enum="";Enabled;Disabled
+	// +kubebuilder:validation:Required
+	// +kubebuilder:default=Disabled
```

Comment:
To use a default the field must be optional and have `omitempty` in the json tag, if you update this then you can remove the empty string case

You also need a second tag, since multiple tools rely on different tags for understanding the defaults
```
// + default:=Disabled
```

Reply:
I had to play with the syntax but it seems that `// +default="Disabled"` does the trick and generates the correct openapi.json file. As far as I tested now everything works, and retrieving the CRD immediately after reboot before any change shows this:
```
    defaultNetwork:
      ovnKubernetesConfig:
        egressIPConfig: {}
        gatewayConfig:
          ipv4: {}
          ipv6: {}
          routingViaHost: false
        genevePort: 6081
        ipsecExternal:
          state: Disabled
        mtu: 1400
```
I hope everything is OK now ;-)